payments-ui-core/src/main/assets/lpms.json

```diff
@@ -456,7 +456,10 @@
   {
     "type": "cashapp",
     "async": false,
-    "fields": []
+    "fields": [],
+    "selector_icon": {
```

Comment:
Any reason SVG wasn't added? Doesn't Android use SVG?

Reply:
We only use the PNG for the remote icons (see [here](https://github.com/stripe/stripe-android/blob/master/payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SelectorIcon.kt)).